docs/contributing.rst

```diff
@@ -39,22 +61,10 @@ SecureDrop's code is written in `Python`_.
 .. _`Ansible`: https://github.com/ansible/ansible
 .. _`Python`: https://github.com/freedomofpress/securedrop/search?l=python
 
+If you are a novice programmer, you can start with issues tagged `"good first
```

Comment:
I know this isn't your wording so feel free to ignore, but I prefer your wording above ("The ``good first issue`` tag is a good introduction to work available in both...") because it's probably a good idea for even expert programmers to start with good first issues to dip their toes in the water.

Reply:
Done in 053651e3f230a67672b56e04ead9a8812e281dae.